jlm/tooling/Command.cpp

```diff
@@ -470,6 +470,28 @@ JlmOptCommand::PrintAsMlir(
 #endif
 }
 
+void
+JlmOptCommand::PrintAsRvsdgTree(
+    const llvm::RvsdgModule & rvsdgModule,
+    const util::filepath & outputFile,
+    util::StatisticsCollector &)
+{
+  auto & rootRegion = *rvsdgModule.Rvsdg().root();
```

Comment:
rootRegion is only used once, so why not inline it?

Reply:
I had it inlined before, but then moved it out because of readability. Subjective preference. I will leave it as it is.